<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ListWidgetEditor</class>
 <widget class="QDialog" name="ListWidgetEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>&amp;Item Properties</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_2_2" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="previewButton" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Colu&amp;mn</string>
        </property>
        <property name="buddy" >
         <cstring>columnsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&amp;Text:</string>
        </property>
        <property name="buddy" >
         <cstring>itemTextLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QToolButton" name="browseButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>&amp;Pixmap:</string>
        </property>
        <property name="buddy" >
         <cstring>browseButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QSpinBox" name="columnsSpinBox" />
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="itemTextLineEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="7" column="1" >
      <widget class="QPushButton" name="moveItemLeftButton" >
       <property name="text" >
        <string>Move Left</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QPushButton" name="moveItemDownButton" >
       <property name="text" >
        <string>Move Down</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="delteItemButton" >
       <property name="text" >
        <string>&amp;Delete Item</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="newItemButton" >
       <property name="text" >
        <string>&amp;New Item</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QPushButton" name="moveItemRighttButton" >
       <property name="text" >
        <string>Move Right</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QPushButton" name="moveItemUpButton" >
       <property name="text" >
        <string>Move Up</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="newSubItemButton" >
       <property name="text" >
        <string>New &amp;Subitem</string>
       </property>
      </widget>
     </item>
     <item rowspan="8" row="0" column="0" >
      <widget class="QTreeWidget" name="treeWidget" />
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QFrame" name="horizontalLine" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
